### PR TITLE
fix missing props and typings for tag component

### PR DESCRIPTION
**Pull request by Sulaymon333**
#### Do Jest tests follow these best practices?

- [ ] `screen` is used for querying.
- [ ] The correct query is used. (Refer to [this list of queries](https://testing-library.com/docs/queries/about/#priority))
- [ ] `asFragment()` is used for snapshot testing.

### CODE REVIEW

**Review by britt6612 — APPROVED** [review]
Thanks for cleaning this up

File: src/js/components/Tag/propTypes.js [hunk 1]
```diff
@@ -1,11 +1,8 @@
 import PropTypes from 'prop-types';
-import { genericProps } from '../../utils/general-prop-types';
 
 let PropType = {};
 if (process.env.NODE_ENV !== 'production') {
   PropType = {
-    ...genericProps,
```
Comment (taysea) [inline src/js/components/Tag/propTypes.js:7]: Why do we remove genericProps / as here? If so, should we be spreading Box prop types?

**Comment by britt6612** [comment]
https://github.com/grommet/grommet-site/pull/504 documentation for PR

File: src/js/components/Tag/propTypes.js (hunk 1 above)
Comment (taysea) [reply src/js/components/Tag/propTypes.js:7]: Since Tag is built on box, it will inherit the props. Resolving this comment.

**Review by taysea — APPROVED** [review]
LGTM.

**Review by jcfilben — APPROVED** [review]
Looks good!

**Pull request merged**